constants.cs

```diff
@@ -1,75 +1,79 @@
 using System;
 
-// Contains all constants for our program. Put any large and/or repeated strings here.
-public static class Constants {
-	public const string USAGE_HELP = "Usage: mono YµP.exe <µPascal filename> <output filename>";
+// All token IDs (add more as we come across them)
+public enum TOKENS {
```

Comment:
All changes relating to TOKENS and Constants class were just to move the TOKENS enum to the correct location (above the class and not inside of it; which was throwing some errors)